docs/installation.rst

```diff
@@ -117,6 +119,8 @@ The `devstack <https://github.com/edx/devstack>`_ install based on docker is a v
             exit
             make studio-restart
 
+    .. note::
+        In versions starting from to the open-release/juniper of the codebase the ``lms.env.json`` or ``cms.env.json`` file have been moved to ``/edx/etc/lms.yaml`` and ``/edx/etc/studio.yaml``
```

Comment:
@luismorenolopera could you please verify if this files exist on your "juniper" version?

Reply:
is /edx/etc/lms.yml and /edx/etc/studio.yml